src/TMS/dev/RadiatorFan.cpp

```diff
@@ -8,10 +8,16 @@ RadiatorFan::RadiatorFan(IO::PWM& pwm): pwm(pwm) {
 }
 
     void RadiatorFan::setSpeed(uint16_t speed) {
-        if (speed > MAX_SPEED) {
-            speed = MAX_SPEED;
+        if (speed == 0){
+            gpio.writePin(EVT::core::IO::GPIO::State::LOW);
         }
-        pwm.setDutyCycle(speed);
+        else{
```

Comment:
It looks like the autoformatter didn't run on this branch. Make sure it does before you merge